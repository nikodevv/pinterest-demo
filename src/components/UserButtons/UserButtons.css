.userButtonsContainer {
  position: fixed;
  bottom: 1rem;
  right: 1rem;
  display: flex;
  flex-direction: column;
}
