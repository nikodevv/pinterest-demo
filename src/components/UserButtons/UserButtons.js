import React, { Component } from 'react';
import './UserButtons.css';
import { FaRegQuestionCircle } from 'react-icons/fa';
import {configure} from "enzyme/build";
import Adapter from "enzyme-adapter-react-16/build";

configure({adapter: new Adapter()});

class UserButtons extends Component {
  render() {
    return <div className="userButtonsContainer">
      <a href='https://github.com/nikodevv/pinterest-demo' target={'_blank'}>
        <FaRegQuestionCircle size="2rem" color="#333"/>
      </a>
    </div>;
  }
}

export default UserButtons;
