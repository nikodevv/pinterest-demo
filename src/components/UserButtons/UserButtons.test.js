import React from 'react';
import { shallow } from 'enzyme';
import UserButtons from './UserButtons';

describe('<UserButtons />', () => {
  test('renders', () => {
    const wrapper = shallow(<UserButtons />);
    expect(wrapper).toMatchSnapshot();
  });
});
