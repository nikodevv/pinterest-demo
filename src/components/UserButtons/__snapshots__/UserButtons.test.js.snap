// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserButtons /> renders 1`] = `ShallowWrapper {}`;
